src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -1347,7 +1322,15 @@ internal abstract class WmiClassBase
         protected static string GetLanguageName(uint? language)
         {
             if (language != null)
```

Comment:
Can we add another condition to check `language >= 0`. This will avoid a change of `ArgumentOutOfRangeException`. Then the only exception left to catch is `CultureNotFoundException`.

Reply:
Done.
Also the parameter was renamed to lcid.